.github/pull_request_template.md

```diff
@@ -1,36 +1,27 @@
 <!-- New Contributor? Welcome!
 
 We recommend you check your privacy settings, so the name and email associated with
-the commit are what you want them to be. See the contribution guide at
+the commits are what you want them to be. See the contribution guide at
 https://github.com/lucyparsons/OpenOversight/blob/develop/CONTRIB.md#recommended-privacy-settings for more infos.
 
 Also make sure you have read and abide by the code of conduct:
 https://github.com/lucyparsons/OpenOversight/blob/develop/CODE_OF_CONDUCT.md
 
+If this pull request is not ready for review yet, please submit it as a draft.
 -->
-
-## Status
-
-Ready for review / in progress
+## Fixes issue
+ <!-- LINK YOUR ISSUE HERE -->
 
 ## Description of Changes
 
-Fixes # .
-
-Changes proposed in this pull request:
-
- -
- -
- -
```

Comment:
This space feels redundant given that we already have the `Description of Changes` section. I've found myself deleting it for every PR I've had recently.